cpu/cc26xx-cc13xx/rf-core/ieee-mode.c

```diff
@@ -1101,6 +1101,8 @@ off(void)
 
   while(transmitting());
 
+  /* stopping the rx explicitly results in lower sleep-mode power usage */
+  rx_off();
```

Comment:
I did test this and I actually did find that the rx_off() is necessary to get the full power reduction. You're logic seems like it should be correct however so maybe I did something wrong in my tests. I can try it again when I have some more time.